projects/pytest.org/package.yml

```diff
@@ -0,0 +1,28 @@
+distributable:
+  url: https://github.com/pytest-dev/pytest/archive/refs/tags/{{ version }}.tar.gz
+  strip-components: 1
+
+versions:
+  github: pytest-dev/pytest
+
+dependencies:
+  python.org: 3.7
+
+build:
+  dependencies:
+    python.org: 3.7
+    pip.pypa.io: 22.3
+  script: |
+    python-venv.sh {{prefix}}/bin/pytest
+
+provides:
+  - bin/pytest
+
+test:
+  dependencies:
+    python.org: 3.7
+    pip.pypa.io: 22.3
```

Comment:
Only need to define python dep at the top level (the other two are redundant).

Python should likely be >=3.7 and pip probably '*'.

Otherwise, it looks good.

Reply:
suggestions applied.